src/client.ts

```diff
@@ -50,6 +50,9 @@ export class DelphinusContract {
   }
 
   async getPastEventsFromSteped(fromBlock: number, toBlock: number, step: number) {
+    if(fromBlock > toBlock) {
+      throw("Error: fromBlock > toBlock");
```

Comment:
Please show the number of  fromBlock and toBlock so it will be convenient for investigation when error happen.

Reply:
Ok